lsql/judge/tests/test_models.py

```diff
@@ -85,6 +85,92 @@ def test_problem_collection_stats(self):
         self.assertEqual(collection.num_solved_by_user(user1), 1)
         self.assertEqual(collection.num_solved_by_user(user2), 0)
 
+    def test_podium(self):
+        """Test the correct performance of the podium"""
+        collection = Collection(name_md='ABC', description_md='blablabla')
+        collection.clean()
+        collection.save()
+        self.assertTrue('ABC' in str(collection))
```

Comment:
Mensaje global para @Tamiflu9 y @Dashito14 también.

Evitad los `assertTrue` siempre que se pueda porque luego los mensajes que muestran cuando no se cumplen son 
"Trues is not False". En https://docs.python.org/3/library/unittest.html#assert-methods podéis ver varios listados con asertos específicos: utilizad el más concreto que se aplique a vuestro caso.

En esta línea lo mejor sería `assertIn`